### ISSUE TITLE
Can you build another release with everything updated as the source?

### ISSUE THREAD

**Issue by almitiger** [angelobreuer/localtunnel.net#25]

**Comment by angelobreuer** [comment]
Yes, I got a report in issue #21 stating the same. I myself did not encounter this issue even with long running tunnels. I provided a possible TCP keep alive fix but got no response since then. I will diagnose the issue further when I return from vacation.

**Comment by almitiger** [comment]
Thanks for the reply, previously I noticed this branch with the fix, but since I'm not good with Visual Studio and in general languages like C#, I couldn't compile the fix due to some errors and missing packages. That is why I ask you, when you can, to compile an updated release that can be of help to more people
